drivers/gles3/shaders/scene.glsl

```diff
@@ -42,7 +42,11 @@ layout(location=5) in vec2 uv2_attrib;
 uniform float normal_mult;
 
 #ifdef USE_SKELETON
+#ifdef PLATFORM_IOS
+layout(location=6) in vec4 bone_indices; // attrib:6
+#else
 layout(location=6) in ivec4 bone_indices; // attrib:6
```

Comment:
@reduz had a spark of genius: instead of adding all these iOS-specific checks, you should likely just `ivec4` to `uvec4` on this line, since it's bound as a `GL_UNSIGNED_SHORT`.